src/Data/Group.hs

```diff
@@ -1,14 +1,32 @@
+{-# LANGUAGE CPP           #-}
+{-# LANGUAGE TypeOperators #-}
```

Comment:
Is `TypeOperators` only needed for the `GHC.Generics` instances? If that's the case, could it be in a CPP conditional block (with a comment to explain why)

Reply:
Yes, but `-XTypeOperators` landed in 6.8.1 (the `GHC.Generics` types didn't show up until later, which is why their CPP is for GHC 7.x). 6.8.1 is older than oldstable, by your previous remarks in this PR, so is more CPP necessary here? I will happily add it if you insist on it (e.g., you want to preserve Haskell98 compatibility for as long as possible), but just want to check first.

Reply:
I'd rather you add it, please

Reply:
No problem at all. I should be able to push it up later this week.

Reply:
Done. Anything else I can do for you?